compiler/tests/fail/stack_allocation/x86-64/unaligned_sub_offset.jazz

```diff
@@ -0,0 +1,10 @@
+// FIXME: double check
+export fn main (reg u64 i) -> reg u64 {
+  stack u32[4] s;
+  reg ptr u64[2] r;
+
+  r = s[i:4];
```

Comment:
I think it is ok but it should not be an internal error.
internal compilation error in function main:
  stack allocation: unaligned sub offset (r (defined at "tests/fail/stack_allocation/x86-64/unaligned_sub_offset.jazz", line 6 (2-3)))

Reply:
New error message
```
"tests/fail/stack_allocation/x86-64/unaligned_sub_offset.jazz", line 9 (8-9):
compilation error in function main:
stack allocation: the access to array
                  r (defined at "tests/fail/stack_allocation/x86-64/unaligned_sub_offset.jazz", line 8 (2-3))
                  could not be proved to be aligned;
                  if you know what you are doing or want to perform an unaligned access,
                  you can use “#unaligned”
```